pennylane/beta/plugins/default_tensor.py

```diff
@@ -232,7 +243,7 @@ def CRot3(a, b, c):
 
 
 # ========================================================
-#  Arbitrary states and operators
+#  Functions which create matrices
```

Comment:
(Minor suggestion): maybe it would be worth moving the utilities and operations that the device uses into a separate files (e.g. `default_tensor_utils.py` or `default_tensor_operations.py`). This way this one would entirely focus on just the device. If accepted, this suggestion would be applicable for `default.tensor.tf` as well.

Reply:
Good suggestion. I seem to remember wanting to do this before, but someone might have talked me out of it. Still makes sense, so I'll do it now

Reply:
I remember now why we didn't do this. These these matrices/functions are all written using numpy, they will not apply to `default.tensor.tf` (or any other sub-devices).

Reply:
I ended up making a utils file, I found it improved the readability of this file